src/Compilers/Core/Portable/Collections/ImmutableArrayExtensions.cs

```diff
@@ -28,7 +28,7 @@ internal static class ImmutableArrayExtensions
         /// <param name="items">The sequence to convert.</param>
         /// <returns>An immutable copy of the contents of the sequence.</returns>
         /// <remarks>If the sequence is null, this will return an empty array.</remarks>
-        public static ImmutableArray<T> AsImmutableOrEmpty<T>(this IEnumerable<T>? items)
+        public static ImmutableArray<T> ToImmutableArrayOrEmpty<T>(this IEnumerable<T>? items)
```

Comment:
⚠️  This should be updated to handle the case of `default(ImmutableArray<T>)`&rarr;`ImmutableArray<T>.Empty`